tests/system/libraries/WindowsLib.py

```diff
@@ -70,6 +70,7 @@ def taskSwitchToItemMatching(targetWindowNamePattern: _re.Pattern, maxWindowsToT
 	builtIn.log(f"Looking for window: {targetWindowNamePattern}", level="DEBUG")
 	startOfTaskSwitcherSpeech = _tryOpenTaskSwitcher()
 	if startOfTaskSwitcherSpeech is None:
+		builtIn.log(f"No speech for opening the task switcher, trying again", level="DEBUG")

```

Comment:
There's a few f-strings that don't need to be

```suggestion
		builtIn.log("No speech for opening the task switcher, trying again", level="DEBUG")
```

Reply:
Fixed these.